### ISSUE TITLE
Improve vote entry page

### ISSUE THREAD

**Issue by glittle** [glittle/TallyJ-4#48]
- Currently, when doing ballots entry, there are a few steps that need mouse use, not easily keyboard accessible.
- All ballot and vote entry should be doable by keyboard only with Tabbing between inputs and buttons.
- The Location should be chosen once. 
   - It should remain known until logout. 
   - It should be visible (in the top banner) and changeable if needed.

**Comment by glittle** [comment]
@copilot See also #50
